packages/eos-components/src/components/ChannelCard.vue

```diff
@@ -0,0 +1,82 @@
+<template>
+  <b-card>
+    <b-card-header class="pb-2">
+      <div
+        v-if="channel.thumbnail"
+        class="mr-2 thumbnail"
+        :style="thumbnailStyles">
+      </div>
+      <div class="title">
+        {{ channel.title }}
+      </div>
+    </b-card-header>
+    <b-card-text class="pt-2 text-muted">
+      {{ description }}
+    </b-card-text>
+  </b-card>
+</template>
+
+<script>
+  export default {
+    name: 'ChannelCard',
+    props: {
+      channel: {
+        type: Object,
+        required: true,
+      },
+      thumbnail: {
+        type: Boolean,
+        default: false,
+      },
+    },
+    computed: {
+      description() {
+        return this.channel.tagline || this.channel.description;
+      },
+      thumbnailStyles() {
+        return {
+          background: `url(${this.channel.thumbnail})`,
+          backgroundSize: 'cover',
+          backgroundPosition: 'center center',
+        };
+      },
+    },
+  }
+</script>
+
+<style lang="scss" scoped>
+  @import '../styles';
+  @import 'bootstrap/scss/bootstrap';
+  @import 'bootstrap-vue/src/index';
```

Comment:
I will open a PR with a new structure for the SCSS. For now yes, you have to do this. I'm afraid we'll need to rethink the overrides.